core/src/test/scala/org/apache/spark/deploy/history/FsHistoryProviderSuite.scala

```diff
@@ -1705,6 +1705,47 @@ abstract class FsHistoryProviderSuite extends SparkFunSuite with Matchers with P
     provider.stop()
   }
 
+  test("clean up expired event log files that don't exist in listing db") {
```

Comment:
Could you add a test prefix like the following?
```scala
- test("clean up expired event log files that don't exist in listing db") {
+ test("SPARK-41447: clean up expired event log files that don't exist in listing db") {
```

Reply:
Sure, have added the test prefix.

Reply:
> Could you add a test prefix like the following?
> 
> ```scala
> - test("clean up expired event log files that don't exist in listing db") {
> + test("SPARK-41447: clean up expired event log files that don't exist in listing db") {
> ```

@dongjoon-hyun, could you review this too, thanks.